ontotext-graphql-playground-component/src/utils/html-util.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Utility class for dynamically loading external JavaScript and CSS files into a webpage.
+ */
+export class HTMLUtil {
```

Comment:
Not sure why it's called HTMLUtil while the doc says stuff related with resource loading?

Reply:
Rename to ResourceUtil